pennylane/transforms/qcut.py

```diff
@@ -939,6 +940,91 @@ def qcut_processing_fn_mc(
     return qml.math.convert_like(np.mean(expvals), res0)
 
 
+@batch_transform
+def cut_circuit_mc(
+    tape: QuantumTape,
+    shots: int,
+    device_wires: Optional[Wires] = None,
```

Comment:
I see that we did the same for `cut_circuit()`, but why should `device_wires` be optional? I'm not sure what we'd do in the `device_wires=None` case :thinking: